apps/admin-ui/src/components/json-file-upload/patternfly/FileUpload.tsx

```diff
@@ -117,7 +117,8 @@ export const FileUpload: React.FunctionComponent<FileUploadProps> = ({
   const onDropAccepted = (acceptedFiles: File[], event: DropEvent) => {
     if (acceptedFiles.length > 0) {
       const fileHandle = acceptedFiles[0];
-      if (event.type === "drop") {
+      // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+      if (event && event.type === "drop") {
```

Comment:
Perhaps we can use optional-chaining here?
```suggestion
      if (event?.type === "drop") {
```